lib/wrappers/genesisProtocol.ts

```diff
@@ -910,62 +910,62 @@ export interface RedeemEventResult {
 }
 
 export interface GenesisProtocolParams {
-  /**
-   * The percentage of the absolute vote that must be exceeded to result in a win.
-   * Must be between 0 and 100.
-   * Default is 50.
-   */
-  preBoostedVoteRequiredPercentage: number;
-  /**
-   * The time limit in seconds for a proposal to be in an absolute voting mode.
-   * Default is 5184000 (two months).
-   */
-  preBoostedVotePeriodLimit: number;
   /**
    * The time limit in seconds for a proposal to be in an relative voting mode.
-   * Default is 604800 (one week).
+   * Default is 259200 (three days).
    */
   boostedVotePeriodLimit: number;
-  /**
-   * Constant A in the threshold calculation,in Wei. See [[GenesisProtocolWrapper.getThreshold]].
-   * Default is 2, converted to Wei
-   */
-  thresholdConstA: BigNumber.BigNumber | string;
-  /**
-   * Constant B in the threshold calculation. See [[GenesisProtocolWrapper.getThreshold]].
-   * Default is 10
-   */
-  thresholdConstB: number;
   /**
    * A floor on the staking fee which is normally computed using [[GenesisProtocolParams.stakerFeeRatioForVoters]].
    * Default is 0
    */
   minimumStakingFee: number;
   /**
-   * The duration of the quietEndingPeriod, in seconds.
-   * Default is 7200 (two hours)
+   * The time limit in seconds for a proposal to be in an absolute voting mode.
+   * Default is 1814400 (three weeks).
    */
-  quietEndingPeriod: number;
+  preBoostedVotePeriodLimit: number;
+  /**
+   * The percentage of the absolute vote that must be exceeded to result in a win.
+   * Must be between 0 and 100.
+   * Default is 50.
+   */
+  preBoostedVoteRequiredPercentage: number;
   /**
    * Constant A in the calculation of the proposer's reward, in Wei
    * See [[GenesisProtocolWrapper.getRedeemableReputationProposer]].
-   * Default is 5, converted to Wei.
+   * Default is 50, converted to Wei.
    */
   proposingRepRewardConstA: BigNumber.BigNumber | string;
   /**
    * Constant B in the calculation of the proposer's reward.
    * See [[GenesisProtocolWrapper.getRedeemableReputationProposer]].
    * Must be between 0 and 100.
-   * Default is 1.
+   * Default is 50.
```

Comment:
im a bit concerned that this is too big. remember https://github.com/daostack/arc.js/issues/196? arc may need to change to treat it as a percentage right? we will test though. @orenyodfat @fmatan